native_client/alphabet.h

```diff
@@ -17,6 +17,12 @@ class Alphabet {
 public:
   Alphabet(const char *config_file) {
     std::ifstream in(config_file, std::ios::in);
+
+    if (!in) {
+      std::cerr << "Unable to load alphabet file " << config_file << std::endl;
+      abort();
```

Comment:
Likely we would prefer a proper error code and handling by the caller?

Reply:
Ok, but alphabet.h might need redesigning then. Maybe it's doing too much in the constructor and most of that code should be moved to another function?

e.g.

````
Alphabet *alpha = new Alphabet();
alpha->loadFile(filename);
````

Reply:
@dabinat Maybe yes. What's your take @reuben ?

Reply:
Yeah, I had to do the same with the ModelState implementations, moving things to a separate initialization method. Sounds good to me.

Reply:
@dabinat Do you think you can address that?

Reply:
I’d assumed when I posted the PR it was essentially an easy 1-line fix and I haven’t had time to dive deeper yet. I will be able to later this week though.